uniparc-rest/src/test/java/org/uniprot/api/uniparc/controller/UniParcStreamControllerIT.java

```diff
@@ -356,6 +554,23 @@ void streamAllContentType(MediaType mediaType) throws Exception {
                 .andExpect(content().contentTypeCompatibleWith(mediaType));
     }
 
+    @ParameterizedTest(name = "[{index}] contentType {0}")
+    @MethodSource("getContentTypes")
+    void streamByProteomeIdAllContentType(MediaType mediaType) throws Exception {
```

Comment:
Do you think we should extract all proteome stream tests to another IT class UniParcProteomeStreamControllerIT.java?